lib/routes/password.js

```diff
@@ -597,24 +609,28 @@ module.exports = function (
                 db.forgotPasswordVerified(passwordForgotToken)
                   .then(
                     function (accountResetToken) {
-                      return mailer.sendPasswordResetNotification(
-                        passwordForgotToken.email,
-                        {
-                          acceptLanguage: request.app.acceptLanguage,
-                          flowId: flowId,
-                          flowBeginTime: flowBeginTime
-                        }
-                      )
-                      .then(
-                        function () {
-                          return request.emitMetricsEvent('password.forgot.verify_code.completed')
-                        }
-                      )
-                      .then(
-                        function () {
-                          return accountResetToken
-                        }
-                      )
+                      return db.accountEmails(passwordForgotToken.uid)
+                        .then((emails) => {
+                          return mailer.sendPasswordResetNotification(
+                            emails,
```

Comment:
This one was a little tricky and touched a lot of components, but I updated all `mailer.*` to accept `emails, account, opts`.